Cargo.toml

```diff
@@ -31,13 +31,13 @@ reqwest = { version = "^0.11.22", default-features = false, features = [
 ] }
 serde = "^1.0"
 slog = "^2.7.0"
-tempfile = "3.5.0"
+tempfile = "3"
```

Comment:
Personally, I like this better, but the rest of this file use more dots.

Can you just restore this to the way it was?

Reply:
Done!